scripts/deploy.py

```diff
@@ -0,0 +1,57 @@
+from brownie import network, accounts, ActiveUser, DutchAuction, NFT, PublicGoods, SimpleAuction, Token
```

Comment:
Excellent work on this.  Need to do some validations on user and stuff like that.  Going to buy an ENS name and will want this to validate the ENS name owner as well